### PR TITLE
Update node-integration.md

### PR DESCRIPTION
**Pull request by camillobruni**
Add `cd node` to make clear that we're adding remotes to the node directory.

### CODE REVIEW

File: src/docs/node-integration.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Remember to run gclient hooks before compiling.
 
 ```bash
 gclient runhooks
-JOBS=4 make test
+JOBS=72 make test
```
Comment (RReverser) [inline src/docs/node-integration.md:29]: Most users probably don't have 72 cores - should we revert this bit?

File: src/docs/node-integration.md [hunk 2]
```diff
@@ -40,11 +40,13 @@ Fork [V8’s Node.js repository on GitHub](https://github.com/v8/node/) (click t
 Add your both your fork and V8’s fork as remotes to the existing checkout:
 
 ```bash
+cd node
 git remote add v8 http://github.com/v8/node
 git remote add <your-user-name> http://github.com/<your-user-name>/node
 git fetch
 git checkout node-ci-<sync-date>
-git checkout -b make-changes
+export BRANCH_NAME=`date +"%Y-%m-%d"`_fix_name
```
Comment (RReverser) [inline src/docs/node-integration.md:48]: Is this something you intended to include in this PR? The description doesn't mention it.

Comment (camillobruni) [reply src/docs/node-integration.md:48]: yes I added this on purpose to make it cleaner when merging branches in.

File: src/docs/node-integration.md (hunk 1 above)
Comment (camillobruni) [reply src/docs/node-integration.md]: Good point, 
cChanged to `nproc` to be more inclusive.

**Review by RReverser — APPROVED** [review]

**Pull request merged**